apps/nextjs/src/app/layout.tsx

```diff
@@ -5,7 +5,7 @@ import Providers from './Providers';
 const inter = Inter({ subsets: ['latin'] });
 
 export const metadata: Metadata = {
-  title: 'NextJS-powered Plone',
+  title: 'NextJS app powered by Plone',
```

Comment:
Check for other instances of `NextJS` and replace with the official spelling of `Next.js`.

```suggestion
  title: 'Next.js app powered by Plone',
```